views/index.erb

```diff
@@ -129,21 +129,6 @@
                   <% end %>
                 </select>
 
-                <label class="usa-label" for="ip-auth-option">
-                  Inherited Proofing
-                </label>
-                <select name="ip_auth_option" id="ip-auth-option" class="usa-select">
```

Comment:
There are lingering references to this input, can we remove them as well?

https://github.com/18F/identity-oidc-sinatra/blob/48344056a069ba99f4cc426d0356701f208498af/app.rb#L55

https://github.com/18F/identity-oidc-sinatra/blob/48344056a069ba99f4cc426d0356701f208498af/app.rb#L165

https://github.com/18F/identity-oidc-sinatra/blob/48344056a069ba99f4cc426d0356701f208498af/spec/app_spec.rb#L94